sdk/tables/azure-data-tables/inc/azure/data/tables.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "azure/data/tables/dll_import_export.hpp"

```

Comment:
Add a comment here, similar to:
https://github.com/Azure/azure-sdk-for-cpp/blob/303cf68356433cec994597de3305de7912517988/sdk/storage/azure-storage-blobs/inc/azure/storage/blobs.hpp#L4-L7